compiler/rustc_lint/src/lints.rs

```diff
@@ -0,0 +1,51 @@
+use rustc_ast::{LitIntType, LitKind};
+use rustc_errors::fluent;
+use rustc_macros::LintDiagnostic;
+use rustc_span::{Span, Symbol};
+
+// FIXME: `#[derive(LintDiagnostic)] with #[suggestion_*]` does not fully output the suggestion,
+// bug maybe?
+// expected: ^^^^^^ help: use an inclusive range instead: `<suggestion>`
+// actual  : ^^^^^^ help: use an inclusive range instead
```

Comment:
I'm having an issue with `#[derive(LintDiagnostic)]` with `#[suggestion_short]` where it doesn't show the replacement.

Reply:
If using suggestion, as in your manual implementation below, works, then just use #[suggestion(..)] here rather than #[suggestion_short]?

Reply:
It works now, thanks.